src/interfaces/filing-interfaces/filing-interface.ts

```diff
@@ -17,10 +18,11 @@ export interface IncorporationFilingIF {
       identifier: string
     },
     incorporationApplication: {
-      nameRequest?: {
-        nrNumber: string
+      // NB: nameRequest must match schema
+      nameRequest: {
```

Comment:
is `nameRequest` required, even as an empty object?

Reply:
Scratch that, i see `legalType` is required.

Reply:
Yes. This "nameRequest" object is required by the Legal API and must conform to the schema (https://github.com/bcgov/business-schemas/blob/master/src/registry_schemas/schemas/name_request.json).
It's a bit confusing because this is **not** the name request data that we get from namex/nro.

Reply:
ahh okay, I thought only the business legal type was required. Interesting.

Reply:
Looks like we don't need `business.legalType` at all (since we can use the value from `incorporationApplication.nameRequest`, but it seems risky to remove it from all 3 UIs right now so I didn't :)